src/lib/booleans.js

```diff
@@ -0,0 +1,131 @@
+function negate(a) {
+  // return !a - to return opposite of a
+  if (a === true) {
+    return false;
+  }
+  return true;
+};
+
+function both(a, b) {
+  if (a && b === true) {
+    return true;
+  }
+  return false;
+}
+
+function either(a, b) {
+  if (a || b === true) {
+    return true;
+  }
+  return false;
```

Comment:
You could replace this with just this, since you are only looking if one of the other is true you can just assert for that:
```
  return a || b;
```